test/transformations.jl

```diff
@@ -13,10 +13,10 @@ using Test
     @test isa(t, AbstractDataTransform)
     @test isempty(t.mean)
     @test isempty(t.scale)
-    @test isequal(X, Y)
+    @test X == Y
```

Comment:
Any reason to change these?

Reply:
I incorrectly thought that `isequal` did not work for `CuArray`. I have restored it.